### PR TITLE
Creating BuildScript_directory-ldap-api 

### PR DESCRIPTION
**Pull request by santosh653**
Creating build script for directory-ldap-api.

The Build is Successfully built & Tested.,

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Apache Directory Shared 0.9.19:
[INFO]
[INFO] Apache Directory Shared ............................ SUCCESS [ 16.190 s]
[INFO] Apache Directory Shared I18n ....................... SUCCESS [ 12.186 s]
[INFO] Apache Directory Shared ASN.1 ...................... SUCCESS [ 4.339 s]
[INFO] Apache Directory Shared ASN.1 Codec ................ SUCCESS [ 1.174 s]
[INFO] Apache Directory Shared Cursor ..................... SUCCESS [ 2.156 s]
[INFO] Apache Directory Shared Ldap Constants ............. SUCCESS [ 0.968 s]
[INFO] Apache Directory Shared Ldap ....................... SUCCESS [ 26.680 s]
[INFO] Apache Directory Shared LDIF ....................... SUCCESS [ 3.918 s]
[INFO] Apache Directory Shared DSML Parser ................ SUCCESS [ 6.799 s]
[INFO] Apache Directory Shared Ldap Converters ............ SUCCESS [ 6.771 s]
[INFO] Apache Directory Shared JNDI ....................... SUCCESS [ 0.982 s]
[INFO] Apache Directory Shared LDAP Schema ................ SUCCESS [ 4.155 s]
[INFO] Apache Directory Shared LDAP Schema DAO ............ SUCCESS [ 0.246 s]
[INFO] Apache Directory Shared LDAP Schema Loader ......... SUCCESS [ 1.535 s]
[INFO] Apache Directory Shared LDAP Schema Manager ........ SUCCESS [ 14.895 s]
[INFO] Apache Directory Shared All ........................ SUCCESS [ 3.855 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:52 min
[INFO] Finished at: 2021-06-04T08:46:21Z
[INFO] ------------------------------------------------------------------------
ubuntu@santoshkulkarni:~/directory-ldap-api$

### CODE REVIEW

File: d/directory-ldap-api/directory-ldap-api_ubuntu_20.04.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+
+export REPO=https://github.com/apache/directory-ldap-api
+
+if [ -z "$1" ]; then
+  export VERSION="shared-parent-0.9.19"
```
Comment (gururajrkatti) [inline d/directory-ldap-api/directory-ldap-api_ubuntu_20.04.sh:23]: Is it 0.1.0 or 0.9.19 ? Also confirm that it is able to build api-asn1-api-1.0.0-M20.jar

**Review by gururajrkatti — CHANGES_REQUESTED** [review]

**Comment by santosh653** [comment]
@gururajrkatti, The Script is successfully building 1.0.0-M20.jar, Please refer to below results.
===========================
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ api-integ ---
[INFO] Building jar: /home/ubuntu/directory-ldap-api/directory-ldap-api/integ/target/api-integ-1.0.0-M20.jar
[INFO]
[INFO] --- maven-site-plugin:3.3:attach-descriptor (attach-descriptor) @ api-integ ---
[INFO]
[INFO] --- tools-maven-plugin:1.4:verify-legal-files (verify-legal-files) @ api-integ ---
[INFO] Checking legal files in: api-integ-1.0.0-M20.jar
[INFO]
[INFO] --- maven-source-plugin:2.2.1:jar-no-fork (attach-sources) @ api-integ ---
[INFO] Building jar: /home/ubuntu/directory-ldap-api/directory-ldap-api/integ/target/api-integ-1.0.0-M20-sources.jar
[INFO]
[INFO] --- maven-install-plugin:2.4:install (default-install) @ api-integ ---
[INFO] Installing /home/ubuntu/directory-ldap-api/directory-ldap-api/integ/target/api-integ-1.0.0-M20.jar to /home/ubuntu/.m2/repository/org/apache/directory/api/api-integ/1.0.0-M20/api-integ-1.0.0-M20.jar
[INFO] Installing /home/ubuntu/directory-ldap-api/directory-ldap-api/integ/pom.xml to /home/ubuntu/.m2/repository/org/apache/directory/api/api-integ/1.0.0-M20/api-integ-1.0.0-M20.pom
[INFO] Installing /home/ubuntu/directory-ldap-api/directory-ldap-api/integ/target/api-integ-1.0.0-M20-sources.jar to /home/ubuntu/.m2/repository/org/apache/directory/api/api-integ/1.0.0-M20/api-integ-1.0.0-M20-sources.jar
[INFO]
[INFO] --------------< org.apache.directory.api:apache-ldap-api >--------------
[INFO] Building Apache Directory LDAP API Distribution 1.0.0-M20        [32/32]
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- maven-clean-plugin:2.4.1:clean (default-clean) @ apache-ldap-api ---
[INFO]
[INFO] --- maven-remote-resources-plugin:1.2.1:process (default) @ apache-ldap-api ---
[INFO]
[INFO] --- maven-site-plugin:3.3:attach-descriptor (attach-descriptor) @ apache-ldap-api ---
[INFO]
[INFO] --- tools-maven-plugin:1.4:verify-legal-files (verify-legal-files) @ apache-ldap-api ---
[INFO]
[INFO] --- maven-source-plugin:2.2.1:jar-no-fork (attach-sources) @ apache-ldap-api ---
[INFO]
[INFO] --- maven-install-plugin:2.4:install (default-install) @ apache-ldap-api ---
[INFO] Installing /home/ubuntu/directory-ldap-api/directory-ldap-api/distribution/pom.xml to /home/ubuntu/.m2/repository/org/apache/directory/api/apache-ldap-api/1.0.0-M20/apache-ldap-api-1.0.0-M20.pom
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Apache Directory LDAP API 1.0.0-M20:
[INFO]
[INFO] Apache Directory LDAP API .......................... SUCCESS [ 11.165 s]
[INFO] Apache Directory LDAP API I18n ..................... SUCCESS [  6.138 s]
[INFO] Apache Directory LDAP API Utilities ................ SUCCESS [  4.165 s]
[INFO] Apache Directory API ASN.1 Parent .................. SUCCESS [  0.277 s]
[INFO] Apache Directory API ASN.1 API ..................... SUCCESS [  0.901 s]
[INFO] Apache Directory API ASN.1 BER ..................... SUCCESS [  1.134 s]
[INFO] Apache Directory LDAP API Parent ................... SUCCESS [  0.038 s]
[INFO] Apache Directory LDAP API Model .................... SUCCESS [ 18.614 s]
[INFO] Apache Directory LDAP API Codec Parent ............. SUCCESS [  0.042 s]
[INFO] Apache Directory LDAP API Codec Core ............... SUCCESS [  3.564 s]
[INFO] Apache Directory LDAP API Net Parent ............... SUCCESS [  0.033 s]
[INFO] Apache Directory LDAP API Network MINA ............. SUCCESS [  0.412 s]
[INFO] Apache Directory LDAP API Codec Standalone ......... SUCCESS [  0.709 s]
[INFO] Apache Directory LDAP API DSML Parent .............. SUCCESS [  0.032 s]
[INFO] Apache Directory LDAP API DSML Parser .............. SUCCESS [  2.705 s]
[INFO] Apache Directory LDAP API Extras ................... SUCCESS [  0.041 s]
[INFO] Apache Directory LDAP API Extras ACI ............... SUCCESS [  2.035 s]
[INFO] Apache Directory LDAP API Schema Parent ............ SUCCESS [  0.030 s]
[INFO] Apache Directory LDAP API Schema Data .............. SUCCESS [ 10.216 s]
[INFO] Apache Directory LDAP API Client Parent ............ SUCCESS [  0.037 s]
[INFO] Apache Directory LDAP API Client API ............... SUCCESS [  0.908 s]
[INFO] Apache Directory LDAP API DSML Engine .............. SUCCESS [  0.434 s]
[INFO] Apache Directory LDAP API Extras Codec API ......... SUCCESS [  0.460 s]
[INFO] Apache Directory LDAP API Extras Codec ............. SUCCESS [  1.803 s]
[INFO] Apache Directory LDAP API Extras Util .............. SUCCESS [  0.787 s]
[INFO] Apache Directory LDAP API Extras Stored Procedures . SUCCESS [  0.405 s]
[INFO] Apache Directory LDAP API Extras Trigger ........... SUCCESS [  1.140 s]
[INFO] Apache Directory LDAP API Schema Converter ......... SUCCESS [  2.177 s]
[INFO] Apache Directory API All ........................... SUCCESS [  2.836 s]
[INFO] Apache Directory LDAP API Client All ............... SUCCESS [  1.562 s]
[INFO] Apache Directory API Integration Tests ............. SUCCESS [ 14.778 s]
[INFO] Apache Directory LDAP API Distribution ............. SUCCESS [  0.264 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:37 min
[INFO] Finished at: 2021-06-09T05:15:39Z
[INFO] ------------------------------------------------------------------------
ubuntu@santoshkulkarni:~/directory-ldap-api$

**Comment by gururajrkatti** [comment]
Ok, Have you updated the version number in the file ?

**Comment by santosh653** [comment]
@gururajrkatti I have now updated the version correctly.,

**Review by gururajrkatti — APPROVED** [review]
ok

**Pull request merged**